### ISSUE TITLE
NLP Recurrent Neural Network(lstm) prediction always gives constant probability values

### ISSUE THREAD

**Issue by Sourabh-Gothe** [tflearn/tflearn#712]
I am basically using word2vec for representing words as vectors and training the neural network with word vectors as input, 
Code :
model.fit(X_train, trainY, validation_set=(X_test, testY),n_epoch=5,shuffle=True, show_metric=True,
          batch_size=100)
predY=np.array(model.predict(X_test))

Output : 
[[ 0.50843388  0.49156615]
 [ 0.50843388  0.49156615]
 [ 0.50843388  0.49156615]
 ..., 
 [ 0.50843388  0.49156615]
 [ 0.50843388  0.49156615]
 [ 0.50843388  0.49156615]]

**Comment by Ruslanmlnkv** [comment]
It's means your model was overfitting. You can add data in train dataset, edit nn architecture or use dropout layers.

**Comment by Sourabh-Gothe** [comment]
Thanks for the answer it helped me a lot,
